web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/fpoversikt/SvpSak.java

```diff
@@ -26,12 +26,17 @@ public record Periode(LocalDate fom, TilretteleggingType type, BigDecimal arbeid
             }
         }
     }
-    public record OppholdPeriode(LocalDate fom, LocalDate tom, Årsak årsak) {
+    public record OppholdPeriode(LocalDate fom, LocalDate tom, Årsak årsak, OppholdKilde kilde) {
```

Comment:
Er oppholdkilde noe vi trenger eksponere til fpoversikt? 
Nå er det slik at FERIE impliserer Inntektsmelding -> så det kan brukes som default dersom kilde mangler.
På sikt vet vi ikke helt hva vi går for - Hvordan skal vi best ivareta de mulige utviklingene
Tanker @palfi  ?

Reply:
Tenker det er fint for bruker å se hvor oppholdet kom fra

Reply:
Ferie kan allerede nå legges inn av saksbehandler, så kilde kan være noe annet enn IM.